helpfiles/batch.txt

```diff
@@ -0,0 +1,20 @@
+
+batch OPERATION ARGUMENT...
+
+This operation expects another operation and its arguments and runs it without
+user interaction. slackroll will error out if user input would be needed to
+select a specific package version among several. In addition, all other
+interactive operations are done automatically with a predefined and documented
+behavior.
+
+Config files are handled as follows:
```

Comment:
Remove the list header and simply include the following points, literally or reworded.

- On package removal, every .new file pair left behind will be removed.
- On package installation, if a single file exists in a .new file pair, it will be kept without the .new suffix. If both files exist, both files will be kept for manual review later.
- No interactive confirmation prompts will be present.
- No pagination will be used.

Reply:
I opted to include your texts here verbatim.